lib/supavisor/db_handler.ex

```diff
@@ -86,13 +82,19 @@ defmodule Supavisor.DbHandler do
   def handle_event(:internal, _, :connect, %{auth: auth} = data) do
     Logger.debug("DbHandler: Try to connect to DB")
 
-    sock_opts = [
-      :binary,
-      {:packet, :raw},
-      {:active, false},
-      {:nodelay, true},
-      auth.ip_version
-    ]
+    sock_opts =
+      [
+        mode: :binary,
+        packet: :raw,
+        # recbuf: 8192,
+        # sndbuf: 8192,
+        # backlog: 2048,
+        # send_timeout: 120,
+        # keepalive: true,
+        # nopush: true,
+        nodelay: true,
+        active: true
+      ] ++ [auth.ip_version]
```

Comment:
Why use `++/2` there? We can put `auth.ip_version` on top, and it will work without such workaround.